src/Components/Footer.js

```diff
@@ -1,58 +1,325 @@
+                </FormGroup>
+                <FormGroup>
+                  <Button
+                    color="primary"
+                    type="submit"
+                    onClick={() => {
+                      valid();
+                    }}
+                  >
+                    {" "}
+                    Update password{" "}
+                  </Button>
+                </FormGroup>
+              </Form>
+            </TabPane>
+            <TabPane tabId="2">
+              <Form style={{ "margin-top": "5px" }} onSubmit={updateUsername}>
+                <FormGroup>
+                  <Input
+                    type="text"
+                    name="new_username"
+                    id="new_username"
+                    placeholder="new username"
+                    value={values.new_username}
+                    onChange={(e) => {
+                      handleChange(e);
+                    }}
+                  />
+                </FormGroup>
+                <FormGroup>
+                  <Button type="submit" color="primary">
+                    Update username
+                  </Button>
+                </FormGroup>
+              </Form>
+            </TabPane>
+          </TabContent>
+        </ModalBody>
+      </Modal>
+
+      <div className="container pt-3 pb-3">
+        <div className="row justify-content-center text-center">
+          <div className="col-6 col-sm-3 d-flex justify-content-around  align-items-center">
+            <a href="https://github.com/code-gambit" target="_blank" className="text-secondary">
+              <span className="fa fa-github fa-lg"></span>
+            </a>
+            <a href="#" target="_blank" className="text-info">
+              <span className="fa fa-linkedin fa-lg"></span>
+            </a>
+            <a href="#" target="_blank" className="text-primary">
+              <span className="fa fa-twitter fa-lg"></span>
+            </a>
+            <a href="#" target="_blank" className="text-danger">
+              <span className="fa fa-instagram fa-lg"></span>
+            </a>
+          </div>
+          {authState.auth.PK ? (
+            <div className="col-4 col-sm-2 d-flex justify-content-around  align-items-center">
+              <span onClick={toggleLightMode}>
+                {!isDarkMode ? (
+                  <span className="fa fa-moon-o fa-lg"></span>
+                ) : (
+                  <span className="fa fa-sun-o fa-lg"></span>
+                )}
+
+              </span>
+                <span onClick={() => {
+                  toggleMain();
+                }} className="fa fa-cog fa-lg"></span>
+
+
+                <span
+                  className="fa fa-sign-out fa-lg "
+                  onClick={handleLogout}
+                ></span>
+
             </div>
+          ) : (
+            null
+          )}
+          <div className="col-12 col-sm-4 d-flex  align-items-center justify-content-center">v-transfer-demo@gmail.com</div>
```

Comment:
Change this using email tag [Can refer to this](https://www.tutorialspoint.com/html/html_email_links.htm)